app/src/main/java/org/simple/clinic/analytics/MixpanelAnalyticsReporter.kt

```diff
@@ -0,0 +1,39 @@
+package org.simple.clinic.analytics
+
+import com.mixpanel.android.mpmetrics.MixpanelAPI
+import org.simple.clinic.BuildConfig
+import org.simple.clinic.ClinicApp
+import org.simple.clinic.user.User
+
+class MixpanelAnalyticsReporter(app: ClinicApp) : AnalyticsReporter {
+
+  private val mixpanel: MixpanelAPI = MixpanelAPI.getInstance(app, BuildConfig.MIXPANEL_TOKEN)
+
+  override fun setLoggedInUser(user: User, isANewRegistration: Boolean) {
+    synchronized(mixpanel) {
+      val userId = user.uuid.toString()
+
+      if (isANewRegistration) {
+        mixpanel.alias(userId, null)
+        mixpanel.identify(userId)
```

Comment:
Can `mixpanel.identify(userId)` move out of the if block altogether?

Reply:
Yes, it can! I'll do this.